eZ/Publish/Core/FieldType/Validator/DestinationContentValidator.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\FieldType\Validator;
+
+use eZ\Publish\API\Repository\ContentService;
+use eZ\Publish\API\Repository\Exceptions\NotFoundException;
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use eZ\Publish\Core\FieldType\ValidationError;
+
+/**
+ * Validator for checking existence of destination content.
+ */
```

Comment:
```suggestion
/**
 * Validator for checking existence of a Content.
 *
 * @internal
 */
```

Reply:
Wouldn't internal gives us a warning when injecting it as DI?